src/main/java/io/github/syst3ms/skriptparser/expressions/ExprStringSplitJoin.java

```diff
@@ -0,0 +1,107 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Main;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+/**
+ * Joins multiple strings together using a delimiter,
+ * splits a given string using a delimiter,
+ * or splits that string in multiple string of a certain length.
+ * Note that for the latter, the last string may contain less characters that specified.
+ *
+ * @name Join/Split
+ * @type EXPRESSION
+ * @pattern (concat[enate]|join) %strings% [(with|using|by) [[the] delimiter] %string%]
+ * @pattern (split %string%|%string% split) (at|using|by) [[the] delimiter] %string%
+ * @pattern (split %string%|%string% split) (with|using|by) %number% [char[acter][s]]
+ * @since ALPHA
+ * @author Mwexim
+ */
+public class ExprStringSplitJoin implements Expression<String> {
+
+	static {
+		Main.getMainRegistration().addExpression(
+				ExprStringSplitJoin.class,
+				String.class,
+				false,
+				"(concat[enate]|join) %strings% [1:[(with|using|by) [[the] delimiter] %string%]]",
+				"(split %string%|%string% split) (at|using|by) [[the] delimiter] %string%",
+				"(split %string%|%string% split) (with|using|by) %number% [char[acter]][s]");
```

Comment:
"every" feels like a natural way to express this, the syntax should allow it

Reply:
Still.

Reply:
I added this? (`"(concat[enate]|join) [(all|every)] %strings% [1:[(with|using|by) [[the] delimiter] %string%]]"`)
If this is not what you meant, can you be more descriptive?

Reply:
I meant something like `split "string" every 2 characters`